WcaOnRails/db/migrate/20190601105825_add_fields_to_regional_organizations.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class AddFieldsToRegionalOrganizations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :regional_organizations, :email, :string, null: false
+    add_column :regional_organizations, :address, :string, null: false
+    add_column :regional_organizations, :directors_and_officers, :text, null: false
+    add_column :regional_organizations, :area_description, :text, null: false
+    add_column :regional_organizations, :past_and_current_activities, :text, null: false
+    add_column :regional_organizations, :future_plans, :text, null: false
```

Comment:
The `NOT NULL` constraint will make existing records invalid, but I guess it's fine given you fill all that info later.

Reply:
Indeed, I'll address this right after merging and deploying the PR.